### DISCUSSION TITLE
Junior Curriculum

### DISCUSSION THREAD

**Discussion by matt-d-webb** [chess-centre/platform#455] [Ideas]
Here is where we should discuss specific chess topics which we want to provide lesson plans for. We can also decide on the structure, including merit badgets or tiers of accomplishment `pawn -> knight -> bishop -> rook -> queen -> king`

@CatVarley

**Comment by CatVarley** [comment]
Loving the merit badges idea. 
Topic wise - For more experienced players is it worth doing blocks of work on specific players/styles of play?
This way in a term you could cover openings, end games, and specific pieces and maneuvers across the term. Be a nice way to block planning. (In a teacher way - if not a chess fan way)

**Reply by allanlealluz** [reply]
hm
